src/App.js

```diff
@@ -9,16 +9,26 @@ export class App extends Component {
     movies: data,
   };
 
+  addMovie = (movie) => {
+    if (this.state.movies
+      .every(presentMovie => presentMovie.imdbId !== movie.imdbId)) {
```

Comment:
```suggestion
      .some(presentMovie => presentMovie.imdbId == movie.imdbId)) {
```
Зачем проверять что у КАЖДОГО не такой id, если нам нужно найти первое только совпадение?